### PR TITLE
Use qe-pipeline-library

### PR DESCRIPTION
**Pull request by psturc**
Testing pipeline is now using bits from [qe-pipeline-library](https://github.com/aerogear/qe-pipeline-library/pull/1)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -139,55 +131,31 @@ pipeline {
...
     }
     post {
-        always{
-            script {
-                sh """
-                oc delete project ${env.OPENSHIFT_PROJECT_NAME}
-                rm -rf ${env.CLONED_REPOSITORY_PATH}
-                """
-            }
-        }
```
Comment (psturc) [inline Jenkinsfile:190]: this part was replaced by [this step](https://github.com/aerogear/unifiedpush-operator/pull/30/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R64)

File: Jenkinsfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ pipeline {
 
     libraries {
         lib('fh-pipeline-library')
+        lib('qe-pipeline-library@AEROGEAR-9314')
```
Comment (psturc) [inline Jenkinsfile]: TODO: delete `@AEROGEAR-9314` before merging

**Review by b1zzu — APPROVED** [review]

**Review by grdryn — APPROVED** [review]
Needs a rebase & the jenkins job should probably pass before merging :sweat_smile:

**Comment by psturc** [comment]
Tests passed! Quickly merging now while it's still working :D

**Pull request merged**